comp/dogstatsd/server/enrich.go

```diff
@@ -28,9 +28,11 @@ var (
 // enrichConfig contains static parameters used in various enrichment
 // procedures for metrics, events and service checks.
 type enrichConfig struct {
-	metricPrefix              string
-	metricPrefixBlacklist     []string
-	metricBlocklist           blocklist
+	// TODO(remy): this metric prefix / prefix blocklist
+	// is independent from the metric name blocklist, that's
+	// confusing and should be merged in the same implemnetation instead.
```

Comment:
It is confusing, but it's different: `metricPrefixBlacklist` does not block metrics, it blocks addition of  `metricPrefix`.

Reply:
Right